pop_tools/grid.py

```diff
@@ -391,3 +391,128 @@ def _compute_corners(ULAT, ULONG):
     corner_lon[0, :, 3] = corner_lon[1, :, 3] - (corner_lon[2, :, 3] - corner_lon[1, :, 3])
 
     return corner_lat, corner_lon
+
+
+@jit(cache=True, nopython=True)
+def dzu_from_dzt(dzt):
+    var = dzt
+    dimz, dimy, dimx = dzt.shape
+    outvar = np.full_like(dzt, fill_value=0)
+
+    for k in np.arange(dimz):
+        for j in np.arange(dimy - 1):
+            for i in np.arange(dimx - 1):
```

Comment:
Tell Numba that the loops can be parallelized:

```suggestion
    for k in prange(dimz):
        for j in prange(dimy - 1):
            for i in prange(dimx - 1):
```